app/tests/test_prompt_templates.py

```
@@ -0,0 +1,218 @@
+"""Tests for enhanced prompt templates."""
+import pytest
```

Ruff (0.11.9)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)